usagov/usagov-2021
# Usagov 1286 add ceo textanchor

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1286

## Description

Added anchor to text "Other ways to contact elected officials" and "Otras formas para contactar a los funcionarios electos" and removed bolding from text on the CEO results page.
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Goto http://localhost/es/funcionarios-electos and http://localhost/elected-officials
- On each page verify that "Other ways to contact elected officials" and "Otras formas para contactar a los funcionarios electos" each have an anchor id.
- Fill out the form for each. Verify that the results page has only the text "Federal Officials", "State Officials", and "State Officias" in bold.

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
